CHANGELOG.md

```diff
@@ -84,6 +84,7 @@ Here is a template for new release sections
 - Fix RTD for emissions (#735)
 - Hot fix: Parameters to be defined as timeseries in `storage_*.csv` (#720)
 - Tests for `E4.minimal_renewable_share_test` (#714, #706)
+- Fixed- Set nans to zero
```

Comment:
You need to pull dev, and then this entry has to be moved into the [Unreleased] section. I think it should not be added to the `fixed` heading but the `added`:
```suggestion
- Added pre-processing step to set NaNs occurring in provided timeseries to zero (#746)
```